community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/MuninnPageCursor.java

```diff
@@ -37,6 +39,9 @@
     private static final boolean tracePinnedCachePageId =
             flag( MuninnPageCursor.class, "tracePinnedCachePageId", false );
 
+    private static final boolean usePreciseCursorErrorStackTraces =
```

Comment:
I was thinking of turning them off by default, or maybe have it off in EE. It might produce a slight bit of overhead, but I don't know if its noticeable.